picard/ui/options/interface.py

```diff
@@ -113,4 +199,130 @@ def starting_directory_browse(self):
             path = os.path.normpath(unicode(path))
             item.setText(path)
 
+    def _valid_action(self, name):
+        return name in self.TOOLBAR_BUTTONS or name == 'separator'
+
+    def _get_icon_from_name(self, name):
+        return self.TOOLBAR_BUTTONS[name]['icon']
+
+    def _insert_item(self, action, index=None):
+        list_item = ToolbarListItem(action)
+        list_item.setToolTip(_(u'Drag and Drop to re-order'))
+        if action in self.TOOLBAR_BUTTONS:
+            list_item.setText(_(self.TOOLBAR_BUTTONS[action]['label']))
+            list_item.setIcon(icontheme.lookup(self._get_icon_from_name(action), icontheme.ICON_SIZE_MENU))
+        else:
+            list_item.setText(self.SEPARATOR)
+        if index:
+            self.ui.toolbar_layout_list.insertItem(index, list_item)
+        else:
+            self.ui.toolbar_layout_list.addItem(list_item)
+        return list_item
+
+    def _all_list_items(self):
+        return [self.ui.toolbar_layout_list.item(i).action_name
+                for i in range(self.ui.toolbar_layout_list.count())]
+
+    def _added_actions(self):
+        actions = self._all_list_items()
+        actions = filter(lambda x: x != 'separator', actions)
+        return set(actions)
+
+    def _current_item(self, return_type='item'):
+        if return_type == 'item':
```

Comment:
I see only uses of `_current_item('row')` and i'm not fond of using string for `return_type` flag.
Rename to '_current_row()' and simplify. Or even better drop this method totally and call `self.ui.toolbar_layout_list.currentRow()` directly.